native/machine-coding/Dice Game/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Dicee</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower|Lobster" rel="stylesheet">
```

Comment:
import the header as done in [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/native/machine-coding/counter/index.html)

Also, remove the other tags